### PR TITLE
[AppVeyor] More minor changes

### PR DESCRIPTION
**Pull request by Margen67**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -25,42 +25,29 @@ init:
...
 build_script:
-  - cmd: xb build --config=%CONFIGURATION% --target=src\xenia-app --target=tests\xenia-cpu-ppc-tests --target=src\xenia-vfs-dump
+  - xb build --config=%CONFIGURATION% --target=src\xenia-app --target=tests\xenia-cpu-ppc-tests --target=src\xenia-vfs-dump
 
 after_build:
-  - cmd: |
-      7z a xenia-%appveyor_repo_branch%.zip LICENSE %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia.exe %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia.pdb
-      7z a xenia-vfs-dump-%appveyor_repo_branch%.zip LICENSE %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia-vfs-dump.exe %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia-vfs-dump.pdb
+  - |-
+      7z a xenia-%APPVEYOR_REPO_BRANCH%.zip LICENSE %APPVEYOR_BUILD_FOLDER%\build\bin\Windows\%CONFIGURATION%\xenia.exe %APPVEYOR_BUILD_FOLDER%\build\bin\Windows\%CONFIGURATION%\xenia.pdb
```
Comment (gibbed) [inline .appveyor.yml]: There was no reason to change this from `%PLATFORM%`.

Comment (Margen67) [reply .appveyor.yml]: It was due to `Platform` being removed. Should I put it back?

Comment (gibbed) [reply .appveyor.yml]: Removed from what, did AppVeyor remove it in a breaking way?

Comment (Margen67) [reply .appveyor.yml]: I removed it from appveyor.yml. https://github.com/xenia-project/xenia/pull/1495/files#diff-11c909939117928998b102a1fff7d363L30

Comment (gibbed) [reply .appveyor.yml]: Oh, if removing that also removes the variable then that's fine.

Comment (Margen67) [reply .appveyor.yml]: It does.

File: .appveyor.yml [hunk 2]
```diff
@@ -21,46 +23,30 @@ pull_requests:
 
 os: Visual Studio 2019
 
-init:
-  - git config --global core.autocrlf input
```
Comment (gibbed) [inline .appveyor.yml]: Why was this removed?

Comment (Margen67) [reply .appveyor.yml]: It doesn't do anything. It's a leftover from the appveyor yml reference: https://www.appveyor.com/docs/appveyor-yml/
